controllers/spacerequest/spacerequest_controller_test.go

```diff
@@ -853,7 +852,7 @@ func TestDeleteSpaceRequest(t *testing.T) {
 	err := apis.AddToScheme(scheme.Scheme)
 	require.NoError(t, err)
 	appstudioTier := tiertest.AppStudioTier(t, tiertest.AppStudioTemplates)
-	srNamespace := newNamespace("jane")
+	srNamespace := spacerequesttest.NewNamespace("jane")
```

Comment:
Not your code per se --  can we put "jane" in a const? Seems it's been used in many place, to avoid any typos in the future.

Reply:
Yeah, thanks for pointing this out, it doesn't look very clean, there are usually some strings that can be extracted to const or variable in our testing code, just didn't saw this aligned with the other unit tests for other controllers. 

Or maybe we could put the string into a variable instead of const ? 🤔